sync-android/src/main/java/com/cloudant/sync/sqlite/android/encryption/DPKBean.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * IBM Confidential OCO Source Materials
+ * 
+ * 5725-I43 Copyright IBM Corp. 2006, 2013
+ * 
+ * The source code for this program is not published or otherwise
+ * divested of its trade secrets, irrespective of what has
+ * been deposited with the U.S. Copyright Office.
+ * 
+*/
+
+/**
+ * This class provides methods for saving the encrypted DPK key into the Android Preference
+ * local storage.
+ */
+
+package com.cloudant.sync.sqlite.android.encryption;
+
+import com.cloudant.sync.sqlite.android.encryption.jackson.JacksonSerializedJSONObject;
```

Comment:
I guess this code came in from JSONStore or somewhere so we need to document that because it looks a bit different from our other classes. I also would like to understand why we need a tonne of jackson/json utilities. I think it should be possible to re-write the (de)serialisation code in this class to use our standard jackson classes without bringing in these utilities.